examples/beop.amp.html

```diff
@@ -2,120 +2,68 @@
   <script async src="https://cdn.ampproject.org/v0.js"></script>
 </head>
 <body>
 
-  <h2>BeOpinion</h2>
+  <h2>BeOp</h2>
 
   <h3>Simple Question</h3>
-  <amp-beopinion width="375" height="1"
+  <amp-beop width="375" height="1"
       layout="responsive"
       data-account="5a8af83d46e0fb0001720141"
       data-content="5a8c383c46e0fb00017ef05a">
-  </amp-beopinion>
-
-  <h3>Question with Tweet</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c504fc9e77c00018d1f1a">
-  </amp-beopinion>
-
-  <h3>Question with Instagram post</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c50a646e0fb00018039df">
-  </amp-beopinion>
-
-  <h3>Question with YouTube video</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8d4f0046e0fb00018bc17d">
-  </amp-beopinion>
-
-  <h3>Question with Dailymotion video</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8d7fddc9e77c00019c3f72">
-  </amp-beopinion>
+  </amp-beop>
 
-  <h3>Question with custom video</h3>
+  <h3>Simple Question (legacy tag)</h3>
   <amp-beopinion width="375" height="1"
       layout="responsive"
       data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c5232c9e77c00018d3a3d">
-  </amp-beopinion>
-
-  <h3>Survey</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c3b69c9e77c00018bffeb">
-  </amp-beopinion>
-
-  <h3>Quiz</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c3d1246e0fb00017f2f47">
+      data-content="5a8c383c46e0fb00017ef05a">
   </amp-beopinion>
 
-  <h3>Personality test</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c3ed0c9e77c00018c2d81">
-  </amp-beopinion>
-
-  <h3>Form</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c3f62c9e77c00018c35dc">
-  </amp-beopinion>
-
-  <h3>Game</h3>
-  <amp-beopinion width="375" height="1"
-      layout="responsive"
-      data-account="5a8af83d46e0fb0001720141"
-      data-content="5a8c41c846e0fb00017f7093">
-  </amp-beopinion>
 
   <h2>Intentionally non-existing content</h2>
-  <amp-beopinion width="375" height="1"
+  <amp-beop width="375" height="1"
```

Comment:
none of these seem to be working for me. Maybe I am doing something wrong?
![image](https://user-images.githubusercontent.com/16087874/73693454-1aaa7a80-468b-11ea-8b5d-46dae5c72200.png)